packages/mds-agency-cache/types.ts

```diff
@@ -23,12 +23,12 @@ import {
   VehicleEvent
 } from '@mds-core/mds-types'
 
-export type StringifiedEvent = Stringify<Omit<VehicleEvent, 'telemetry' | 'event_types'>> &
-  Pick<VehicleEvent, 'event_types'>
 export type StringifiedTelemetry = Stringify<Omit<Telemetry, 'gps'>> & {
   gps: Stringify<Omit<TelemetryData, 'charge'>>
 }
-export type StringifiedEventWithTelemetry = StringifiedEvent & { telemetry?: StringifiedTelemetry }
+
+type StringifiedEvent = Stringify<Omit<VehicleEvent, 'telemetry' | 'event_types'>> & Pick<VehicleEvent, 'event_types'>
```

Comment:
Are there plans to use this type for anything else ever? Otherwise, might as well just have `StringifiedEvent` but assigned to the `StringifiedEventWithTelemetry` definition.

Reply:
No plans, but I was keeping `StringifiedEvent` around as a private type just to make the composition a bit easier. This will be totally reworked once we move the agency caching layer to the ingest-service, I think.